src/DotNetStatus/Views/Usage/Index.cshtml

```diff
@@ -5,11 +5,11 @@
 @model Microsoft.Fx.Portability.ObjectModel.UsageDataCollection
 
 @{
-    ViewBag.Title = "API Platform Availability";
+    ViewBag.Title = "ApiPort Usage";
 }
 
 <div class="jumbotron">
-    <h1>.NET API Usage</h1>
+    <h2>.NET Framework API usage data from ApiPort tool</h2>
```

Comment:
The official name for the tool is actually .NET Portability Analyzer (I always forget, but might be good to use that on the site)